lib/vc.js

```diff
@@ -419,10 +419,8 @@ function _checkCredential(credential) {
     throw new Error('"type" property is required.');
   }
 
-  if(!Array.isArray(credential['type']) ||
-      !credential['type'].includes('VerifiableCredential')) {
-    throw new Error(
-      '"type" must include `VerifiableCredential`.');
+  if(!jsonld.getValues(credential, 'type').includes('VerifiableCredential')) {
```

Comment:
There is also a `hasValue(s, p, v)` helper.  (Though the code here might be a bit more performant in this simple use case.)

Reply:
Thanks, good to know.